examples/mongodb/mongo-environment.js

```diff
@@ -1,9 +1,6 @@
-// mongo-environment.js
-const NodeEnvironment = require('jest-environment-node').default;
-
-const path = require('path');
-
-const fs = require('fs');
+import fs from 'fs';
```

Comment:
I don't think this should be ESM?

Reply:
It probably doesn't hurt, tho

Reply:
It was hard to look at `require('jest-environment-node').default` above. That’s the only reason. I can revert.

Reply:
Nah, it's fine. Sorta surprised it works tho - the `default` in CJS is different from native ESM `default`. Can use `const {TestEnvironment} = require('jest-environment-jsdom')`